index.bs

```diff
@@ -4089,6 +4091,14 @@ Transitioning to Recommendation</h4>
 	[=Advisory Committee representatives=] <em class="rfc2119">may</em> initiate an [=Advisory Committee Appeal=]
 	of the decision to advance the technical report.
 
+	The newly published [=Recommendation=]
+	<em class=rfc2119>must not</em> have any [=substantive changes=] to the document
```

Comment:
```suggestion
	<em class=rfc2119>must not</em> include any [=substantive changes=] made to the document
```

Reply:
How about "must not include any substantive changes [ from | compared to ] ..."?

Reply:
that phrasing sounds like it may be permissible to include substantive changes (other than dropping at risk) so long as they haven't been made to the document since CRS. They could, for example, be prepared on the side by the Team in response to AC Review. That wouldn't be appropriate.

How about:
```suggestion
	<em class=rfc2119>must not</em> make any [=substantive changes=] to the document
```
together with
```diff
- since the [=Candidate Recommendation Snapshot=]
+ compared to the [=Candidate Recommendation Snapshot=]
```